### ISSUE TITLE
Few bugs in the new big video  on home page

### ISSUE THREAD

**Issue by para2022** [WWBN/AVideo#8731]
The new big video for the homepage has 2 issues I've identified.

1. Because it mutes itself it also mutes all videos throughout the site when it shouldn't.

2. It is affecting the 'watching now' too (green eye). The big video is affecting the 'watching now'. For every person that has the main page open it will increase the 'watching now' on the last video uploaded. An example is that the new big video displays the last upload, so If you scroll down the page and look at the last video uploaded. It may show 10-20 or whatever it is at the time on the "watching now' count on the last upload, but nobody has actually viewed the video yet and therefore the 'watching now' doesn't match the 'total views' because it is getting its 'watching now' count from the big video because that video is being viewed all the time because its on the homepage. A solution would be remove the 'watching now' function from the big video on homepage. It isn't needed anyway.

To sum up, a simple fix for this would be to stop the big video from muting all vids throughout the site. Secondly to turn off the 'watching now' plugin from the big video which then does not affect the 'watching now' on the last video uploaded and then the 'watching now' and 'total views' would go back to how they should be. Hope this helps.

**Comment by para2022** [comment]
Also could you please turn it off on the category pages? Its not needed there either.

https://ourtube.co.uk/cat/music

**Comment by DanielnetoDotCom** [comment]
Hi, the "mutes all" may be a bug, the rest are not bugs

**Comment by para2022** [comment]
The videos still are on mute throughout the site after updates.

**Comment by DanielnetoDotCom** [comment]
This update should fix the muted thing

**Comment by para2022** [comment]
Thanks mate that has fixed it. Just noticed its affecting the load more on the front page too. Maybe have a look when you get some time. Thank you

**Comment by DanielnetoDotCom** [comment]
Can you please detail the load more issue?

Can you reproduce it on demo site?

**Comment by para2022** [comment]
If you click here https://ourtube.co.uk/ and click on the 'load more' on the front page. It has stopped working again. You did fix this a while ago but it seems the problem is now back again.

**Comment by DanielnetoDotCom** [comment]
Thanks, please check this update then clear the cache

**Comment by para2022** [comment]
Thanks Dan, working again now. Shouldn't watching the big video generate a view, if you've not watched it before? It doesn't behave that way at the moment. Cheers

**Comment by DanielnetoDotCom** [comment]
https://github.com/WWBN/AVideo/wiki/How-Video-Count-Works
